dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -86,3 +89,12 @@ includePathInWASReqURL.desc=Setting the Path parameter can allow the client/brow
 
 trackLoggedOutSSOCookies=Track logged out LTPA single signon tokens
 trackLoggedOutSSOCookies.desc=Specifies whether to track LTPA single signon tokens that are logged out on a server so that it can not be reused on the same server.
+
+overrideHttpAuthenticationMechanism=Name of HttpAuthenticationMechanism to override
+overrideHttpAuthenticationMechanism.desc=Specifies the HttpAuthenticationMechanism for overriding an application-defined HttpAuthenticationMechanism. The HttpAutheMechanism is defined by the Java EE 8 Security specification. The acceptable value is BASIC or FORM. When FORM is used, specify the loginFormURL and loginErrorURL properties.
```

Comment:
Could we mention what HttpAuthenticationMechanism is (like how we say that loginFormURL and loginErrorURL are properties)?

Also, in the second sentence, we say "HttpAutheMechanism" instead of "HttpAuthenticationMechanism." Just wanted to be sure that wasn't a typo. Thanks! :)

Reply:
For "HttpAutheMechanism", it's typo and corrected.
There is not a short explanation what HttpAuthenticationMechanism is, it could be a class, or annotation. Therefore, it states that it's defined by Java EE 8 security.